src/sb_utils/include/RvizUtils.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Created By: Robyn Castro
+ * Created On: November 10th, 2017
+ * Description: Utilities to create Rviz visualisation messages
+ *
+ */
+#ifndef SB_UTILS_RVIZUTILS_H
+#define SB_UTILS_RVIZUTILS_H
+
+// Messages
+#include <geometry_msgs/Point.h>
+#include <visualization_msgs/Marker.h>
+
+using namespace std;
+using namespace visualization_msgs;
```

Comment:
`using namespace` lines should be in the `.cpp` file. This does mean the signatures and fields in the header will need to be changed, but If they're in the header, then anything that includes the header will automatically get them, which can cause conflicts between things with the same name (say if we defined a class called `Marker`, then it would conflict with `visualization_msgs::Marker` if the file using `Marker` included this header)

Reply:
Fixed